src/Sentry.Extensions.Logging/BindableSentryLoggingOptions.cs

```diff
@@ -9,11 +9,22 @@ internal class BindableSentryLoggingOptions : BindableSentryOptions
     public LogLevel? MinimumEventLevel { get; set; }
     public bool? InitializeSdk { get; set; }
 
+    [Experimental(Infrastructure.DiagnosticId.ExperimentalFeature)]
+    public BindableSentryLoggingExperimentalOptions ExperimentalLogging { get; set; } = new();
+
+    [Experimental(Infrastructure.DiagnosticId.ExperimentalFeature)]
```

Comment:
Same again: any reason to mark an internal member Experimental?

Reply:
no ... I was a bit overzealous ...
removed all `ExperimentalAttribute` from `internal` types and members